e2e/test/scenarios/models/models.cy.spec.js

```diff
@@ -231,6 +231,21 @@ describe("scenarios > models", () => {
     H.echartsContainer().should("not.exist");
   });
 
+  it.only("only shows model info modal once when turning a question into a model", () => {
```

Comment:
will remove `only`, i'm currently getting help on why this test is not acting correctly due to a BE error